src/sync-n-parent.js

```diff
@@ -0,0 +1,27 @@
+AFRAME.registerComponent('sync-n-parent',
```

Comment:
If I am to understand the goal of this component, I believe it is:

Give everyone in the scene who joins one of these items and parent it to their skeleton.

Is that right?